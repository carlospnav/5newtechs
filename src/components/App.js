import styled from "styled-components"
import "theme/global"

const Main = styled.div`
  display: flex;
  flex-direction: column;
`

export default ({ children }) => <Main>{children}</Main>
