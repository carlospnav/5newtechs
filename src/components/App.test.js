import { shallow } from "enzyme"
import App from "./App"
import "jest-styled-components"

describe("<App>", () => {


  it("should render an app with its children properly.", () => {
    const wrapper = shallow(
      <App>
        <p>Here comes the app!</p>
      </App>
    )

    expect(wrapper.dive()).toMatchSnapshot()
  })
})