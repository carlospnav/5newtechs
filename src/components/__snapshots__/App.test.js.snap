// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App> should render an app with its children properly. 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

<div
  className="c0"
>
  <p>
    Here comes the app!
  </p>
</div>
`;
